src/table/__tests__/TableHeadWrapper.spec.jsx

```diff
@@ -143,4 +143,104 @@ describe('<TableHeadWrapper />', () => {
 
     expect(handleCellFocus.handleClickToFocusHead).to.have.been.calledOnce;
   });
+
+  it('should has `scope` property set to col', () => {
+    const { queryByText } = render(
+      <TableHeadWrapper tableData={tableData} theme={theme} layout={layout} changeSortOrder={changeSortOrder} />
+    );
+
+    const firstColQuery = queryByText(tableData.columns[0].label).closest('th');
+    const secondColQuery = queryByText(tableData.columns[1].label).closest('th');
+
+    expect(firstColQuery).to.have.attribute('scope', 'col');
+    expect(secondColQuery).to.have.attribute('scope', 'col');
+  });
+
+  it('should has `role` property set to button', () => {
+    const { queryByText } = render(
+      <TableHeadWrapper tableData={tableData} theme={theme} layout={layout} changeSortOrder={changeSortOrder} />
+    );
+
+    const firstColQuery = queryByText(tableData.columns[0].label).closest('th');
+    const secondColQuery = queryByText(tableData.columns[1].label).closest('th');
+
+    expect(firstColQuery).to.have.attribute('role', 'button');
+    expect(secondColQuery).to.have.attribute('role', 'button');
+  });
+
+  it('should has `aria-sort` property', () => {
+    const { queryByText } = render(
+      <TableHeadWrapper tableData={tableData} theme={theme} layout={layout} changeSortOrder={changeSortOrder} />
+    );
+
+    const firstColQuery = queryByText(tableData.columns[0].label).closest('th');
+    const secondColQuery = queryByText(tableData.columns[1].label).closest('th');
+
+    expect(firstColQuery).to.have.attribute('aria-sort', 'ascending');
+    expect(secondColQuery).to.not.have.attribute('aria-sort');
+  });
+
+  it('should has `aria-pressed` property', () => {
```

Comment:
You can make all of these tests into one. no need to re-render every time since there is no difference

Reply:
Sure, I was trying to stick to one of the testing rules which indicated that: "each test case might only check for one thing"

but if you thinking that could be redundant to do at some point, i could combine them as well 👌

Reply:
✅ Fixed